### PR TITLE
GitHub Deployments: Recommend it inside Hosting Config

### PR DESCRIPTION
**Pull request by zaguiini**
Closes https://github.com/Automattic/dotcom-forge/issues/5248

## Proposed Changes

image

### CODE REVIEW

File: packages/components/src/card/style.scss [hunk 1]
```diff
@@ -69,6 +69,10 @@
 	&.is-warning {
 		border-left: 3px solid var(--color-warning-20);
 	}
+
+	&.is-primary {
+		border-left: 3px solid var(--color-accent);
+	}
```
Comment (Aurorum) [inline packages/components/src/card/style.scss]: Sorry if this is a random question! But does this not just make it equivalent to the `Banner` component? Might I be missing something about why that can't be used instead?

Screenshot 2024-02-27 at 23 19 35

Comment (zaguiini) [reply packages/components/src/card/style.scss]: It's a good question!

I tried using the banner component but I found that the icon was getting the accent background and border radius whenever I tried using the GitHub social logo.

I'll try again though; I prefer that approach.

Comment (Aurorum) [reply packages/components/src/card/style.scss]: Ah, I see! You should be able to use the `disableCircle` prop for that. I haven't tested it but I think you could either add a `socialIcon` prop to the component here:

https://github.com/Automattic/wp-calypso/blob/035f94a42eb8d8e30450e6853ec6eef02385e692/client/components/banner/index.jsx#L196-L200

Or pass the GitHub logo as an image with the `iconPath` prop - seems like there's one lying around in Calypso, though not sure how ideal that is: `/calypso/images/sharing/p2-github-logo.svg` (might need to style the size down though)

Comment (zaguiini) [reply packages/components/src/card/style.scss]: Yeah! What I'm doing is modify the banner component to accept a `ReactNode` as well. Thanks for pushing; I will be able to use the banner component indeed!

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/11490310

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @zaguiini for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/11490310).

**Pull request merged**